src/styles/blocs/header.scss

```diff
@@ -0,0 +1,66 @@
+.header {
+  display: flex;
+  height: $header-height;
+  background-color: #fff;
+
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  margin: auto 50px;
+  justify-content: space-between;
+
+  &__logo {
+    display: flex;
+  }
+
+  &__list {
+    display: flex;
+    height: 100%;
+    margin: 0;
+    padding-inline-start: 0;
+  }
+
+  &__list > :not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__item {
+    display: flex;
+    list-style: none;
+  }
+
+  &__link {
+    color: $main-text-color;
+    text-decoration: none;
+    display: flex;
+    align-items: center;
+
+    transition: 0.3s;
+
+    &:hover {
+      color: $hovering-color;
+      transition: 0.3s;
```

Comment:
don't use it like that, because it's the same as transition: all 0.3s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md